docker-compose.yml

```diff
@@ -66,9 +149,14 @@ services:
       - 8080:8080
     volumes:
       - .:/app
-    container_name: try-cb-api-cap
+    container_name: local-capella-backend-python
     profiles:
-      - capella
+      - local-capella
+
+  # ---------------------FRONTENDS---------------------
+  # There are 5 frontend services, all are functionally
+  # identical. The only difference is which profile the
+  # service belongs to, and which backend it depends on
```

Comment:
Hmm. I think this complexity is the reason that I spun some things to mix-and-match.yml instead.